### PR TITLE
fix(data-warehouse): Allow the choice of SSL for mysql connections

### PR DESCRIPTION
**Pull request by Gilbert09**
## Problem
- Adding in mandatory SSL support for mysql connections caused syncs to break for a customer ([see ticket](https://posthoghelp.zendesk.com/agent/tickets/20515))

## Changes
- Updated the UI to let users choose whether they want to use SSL for the connection
- Defaults to `true` everywhere to mimic existing behavior 

## Does this work well for both Cloud and self-hosted?
Yes

## How did you test this code?
Stepped through locally

### CODE REVIEW

File: posthog/temporal/data_imports/workflow_activities/import_data_sync.py [hunk 1]
```diff
@@ -144,6 +144,8 @@ def import_data_activity_sync(inputs: ImportDataActivityInputs):
             ssh_tunnel_auth_type_passphrase = model.pipeline.job_inputs.get("ssh_tunnel_auth_type_passphrase")
             ssh_tunnel_auth_type_private_key = model.pipeline.job_inputs.get("ssh_tunnel_auth_type_private_key")
 
+            using_ssl = str(model.pipeline.job_inputs.get("using_ssl", True)) == "True"
```
Comment (Gilbert09) [inline posthog/temporal/data_imports/workflow_activities/import_data_sync.py:147]: All the job inputs get stringified _somewhere_ and so we have to test against `"True"`

**Review by rossgray — COMMENTED** [review]
Are there any potential security issues here with user data being sent to us without encryption? eg did we enforce SSL for a reason previously?

**Comment by Gilbert09** [comment]
> Are there any potential security issues here with user data being sent to us without encryption? eg did we enforce SSL for a reason previously?

Dont think so - we enabled it for planetscale databases which required it

**Review by rossgray — APPROVED** [review]
Happy to approve if this is how it was working before

**Pull request merged**